2025/using-components-2/demo/2-react/README.md

```diff
@@ -0,0 +1,37 @@
+# Vite + React demo
+
+[Session Description](../..)
+
+The project was created using the [Vite sample app in jsapi-resources](https://github.com/Esri/jsapi-resources/tree/main/component-samples/map-components/samples/vite)
+
```

Comment:
Add a section like https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/main/2025/build-tooling/demo/2-react/README.md#key-changes-from-1-javascript to explain what changed from the first demo?
especially useful since we don't go into installation instructions details in the presentation